node_cli/cli/sync_node.py

```diff
@@ -36,11 +38,31 @@ def sync_node():
     pass
 
 
-@sync_node.command('init', help="Initialize sync SKALE node")
+@sync_node.command('init', help=TEXTS['init']['help'])
 @click.argument('env_file')
+@click.option(
+    '--archive',
+    help=TEXTS['init']['archive'],
+    is_flag=True
+)
+@click.option(
+    '--catchup',
+    help=TEXTS['init']['catchup'],
+    is_flag=True
+)
+@click.option(
+    '--historic-state',
+    help=TEXTS['init']['historic_state'],
+    is_flag=True
+)
 @streamed_cmd
-def _init_sync(env_file):
-    init_sync(env_file)
+def _init_sync(env_file, archive, catchup, historic_state):
+    if (historic_state and not archive):
```

Comment:
`if historic_state and not archive:` - better

Reply:
+